src/main/resources/templates/createSurvey.html

```diff
@@ -28,7 +29,9 @@
     <input class = "CreateSurveyButton"  type="submit" value="Create Question">
 </form>
 
+<button onclick="sendSurvey()">Create Survey</button>
 <button class = "CreateSurveyButton" onclick="sendSurvey()">Create Survey</button>
+<button onclick="back()">Back</button>
```

Comment:
I think this makes it have 2 Create Survey buttons on the survey/create page

Reply:
that create survey button that is says I added shouldn't be there ill fix that thanks